pkg/metrics/datastore.go

```diff
@@ -0,0 +1,25 @@
+package metrics
+
+import (
+	met "github.com/blocknative/dreamboat/metrics"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func InitBadgerMetrics(m *met.Metrics) error {
+	a := map[string]*prometheus.Desc{
+		"badger_v2_blocked_puts_total":   prometheus.NewDesc("badger_blocked_puts_total", "Blocked Puts", nil, nil),
```

Comment:
can we rename these to datastore instead of badger so that its future proof for when we move to a generic data store implementation

Reply:
Renamed